src/example-1.go

```diff
@@ -5,14 +5,40 @@ import (
 	"fmt"
 	"github.com/zeebe-io/zeebe/clients/go/pkg/pb"
 	"github.com/zeebe-io/zeebe/clients/go/pkg/zbc"
+	"os"
 )
 
-const BrokerAddr = "0.0.0.0:26500"
+const ZeebeAddr = "0.0.0.0:26500"
 
+/*
+Sample application that connects to a cluster on Camunda Cloud, or a locally deployed cluster.
+
+When connecting to a cluster in Camunda Cloud, this application assumes that the following
+environment variables are set:
+
+ZEEBE_ADDRESS
+ZEEBE_CLIENT_ID
+ZEEBE_CLIENT_SECRET
+ZEEBE_AUTHORIZATION_SERVER_URL
+
+Hint: When you create client credentials in Camunda Cloud you have the option
+to download a file with above lines filled out for you.
```

Comment:
```suggestion
to download a file with the lines above filled out for you.
```
Or something else, but "with above lines" sounds weird to me.